python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -745,7 +747,7 @@ def generateTractOutputProducts(self, dataRefDict, tract,
 
         return retStruct
 
-    def _computeReferenceOffsets(self, stdCat, bands):
+    def _computeReferenceOffsets(self, stdCat, lutCat, physicalFilterMap, bands):
         """
         Compute offsets relative to a reference catalog.
```

Comment:
When we chatted, you mentioned that this was mostly used for testing. Should that be made clear in the docs? Because I worry that the implementation may now break if someone tries to use it on data that has multiple physical filters mapping to the same band.